README.md

```diff
@@ -111,7 +118,7 @@ After you're finished please remove all the comments and instructions!
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://google.com)
+- [go to my GitHub and try it now](https://github.com/AztkMarty)
```

Comment:
- [ ] kindly consider removing this section as you are not going to deploy this project probably in the future project you will add it once you deployed your project removing it will  avoid confusion to people visiting your readme